lua/entities/gmod_tardis/modules/sh_redecorate.lua

```diff
@@ -22,17 +22,13 @@ local int_saved_data_names = {
 	"security"
 }
 
-local function apply_saved_data(object, data, names)
+function TARDIS:apply_redecoration_data(object, data)
```

Comment:
A local function seemed fine here, why was this changed? Also we do not want to generally define TARDIS globals inside the entity folder